### PR TITLE
Newsletter-57: translate into Chinese

### PR DESCRIPTION
**Pull request by bittomhan**

### CODE REVIEW

**Comment by bittomhan** [comment]
translate "Lightning BOLTs" to "闪电网络规范"
modified all "Bitcoin Improvement Proposals(BIPs)" to "比特币改进提案（BIPs）" and removed those space on both sides

File: _posts/zh/newsletters/2019-07-31-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+  基于对该 PR 的早期反馈，BIP322 还进行了更新，以便使用传统 P2PKH 密钥签名和验证消息时使用旧的 signmessage 格式。这使得 BIP322 工具能够完全向后兼容现有的、广泛实施的 signmessage 标准（该标准仅支持 P2PKH）。
+
+- **<!--bloom-filter-discussion-->****布隆过滤器讨论：**在[上周 Newsletter][Newsletter #56] 的 *值得注意的变更* 部分中，我们提到了一个已合并的 [PR][Bitcoin Core #16152]，该 PR 禁用了 Bitcoin Core 默认配置中的布隆过滤器。PR 的作者在 Bitcoin-Dev 邮件列表中[宣布][bloom announce]了这一未发布的更改，几个人对此发表了问题或担忧。讨论的一些要点包括：
+
+  - **<!--no-urgent-action-required-->***无需紧急行动：*截至撰写本文时，接受入站连接的节点（监听节点）中有超过 20% 运行的是一年以上的 Bitcoin Core 版本 [根据 BitNodes 数据][bitnodes dashboard]。至少 5% 运行的是两年以上的版本。推断到未来，这意味着到 2022 年初，仍将有超过 500 个监听节点提供布隆过滤器支持。因此，即使不采取进一步行动，钱包开发者可能仍有足够的时间来调整他们的程序。
+
+  - **<!--spies-likely-to-run-their-own-nodes-->***间谍可能会运行自己的节点：*除了未升级的节点之外，区块链分析公司可能会继续运行自己的节点，以在未来提供布隆过滤器支持，从而收集钱包的统计数据，这些数据的过滤器[泄露信息][filter privacy]关于它们包含的地址。
+
+  -**<!--dns-seeds-can-return-only-nodes-signaling-bip111-->** *DNS seeds 只能返回信号 BIP111 的节点：*大多数 P2P 轻钱包会查询一个或多个比特币 DNS seeds 以获取要使用的节点列表。一些 seeds 允许根据节点配置的服务比特来筛选返回的节点[^dns-query]，其中支持布隆过滤器的服务位由 [BIP111][] 指定。
```
Comment (bitschmidty) [inline _posts/zh/newsletters/2019-07-31-newsletter.md:37]:
```suggestion
  - **<!--dns-seeds-can-return-only-nodes-signaling-bip111-->** *DNS seeds 只能返回信号 BIP111 的节点：*大多数 P2P 轻钱包会查询一个或多个比特币 DNS seeds 以获取要使用的节点列表。一些 seeds 允许根据节点配置的服务比特来筛选返回的节点[^dns-query]，其中支持布隆过滤器的服务位由 [BIP111][] 指定。
```

File: _posts/zh/newsletters/2019-07-31-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+  -**<!--bip157-would-use-more-bandwidth-than-bip37-->***BIP157 将使用比 BIP37 更多的带宽：*有人建议，一些使用 [BIP37][]布隆过滤器的用户将无法切换到 [BIP157][] 致密区块过滤器，因为后者的带宽使用更高。Optech 在 [Newsletter #43][] 中简要调查了过滤器大小，发现过滤器本身每月使用不到 100 MB。客户端还需要下载匹配的区块，这将根据其活动而有所不同（对于典型用户来说，大约每个发送或接收的交易对应一个区块；大多数区块小于 2 MB）。目前尚不清楚有多少用户会因必须下载这些数据量而受到显著影响。
```
Comment (bitschmidty) [inline _posts/zh/newsletters/2019-07-31-newsletter.md:39]:
```suggestion
  - **<!--bip157-would-use-more-bandwidth-than-bip37-->***BIP157 将使用比 BIP37 更多的带宽：*有人建议，一些使用 [BIP37][]布隆过滤器的用户将无法切换到 [BIP157][] 致密区块过滤器，因为后者的带宽使用更高。Optech 在 [Newsletter #43][] 中简要调查了过滤器大小，发现过滤器本身每月使用不到 100 MB。客户端还需要下载匹配的区块，这将根据其活动而有所不同（对于典型用户来说，大约每个发送或接收的交易对应一个区块；大多数区块小于 2 MB）。目前尚不清楚有多少用户会因必须下载这些数据量而受到显著影响。
```

File: _posts/zh/newsletters/2019-07-31-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    [...]
+    ```
+
+{% include linkers/issues.md issues="800,766,2842,643,16440,16152" %}
+[bech32 series]: /zh/bech32-sending-support/
+[bolt1 error]: https://github.com/lightningnetwork/lightning-rfc/blob/master/01-messaging.md#requirements-2
+[drivechain discussion]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2017-December/015339.html
+[core rc]: https://bitcoincore.org/bin/bitcoin-core-0.18.1/
+[fidelity coinjoin]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-July/017169.html
+[bitnodes dashboard]: https://bitnodes.earn.com/dashboard/
```
Comment (bitschmidty) [inline _posts/zh/newsletters/2019-07-31-newsletter.md:104]: Probably update to

```suggestion
[bitnodes dashboard]: https://bitnodes.io/
```

**Review by bitschmidty — CHANGES_REQUESTED** [review]

**Pull request merged**